tests/test_parser.py

```diff
@@ -533,10 +535,28 @@ def test_elasticsearch_flag(self):
                 "pid": "number",
                 "string_array": ["string"],
                 "obj_array": ["string"],
-                "process": {"name": "string"}
+                "opcode": "number",
+                "process": {"name": "string"},
+                "unique_pid": "string"
+            },
+            "file": {
+                "opcode": "number",
+                "unique_pid": "string"
             }
         })
 
+        with elasticsearch_syntax:
```

Comment:
can you confirm that these tests always raise `EqlSyntaxError` when elasticsearch_syntax is disabled?